builtins/amp-video.js

```diff
@@ -49,7 +73,7 @@ export function installVideo(win) {
         assertHttpsUrl(this.element.getAttribute('src'), this.element);
       }
       this.propagateAttributes(
-          ['src', 'controls', 'autoplay', 'muted', 'loop', 'poster'],
+          ['src', 'controls', 'autoplay', 'muted', 'loop', 'poster', 'preload'],
```

Comment:
Does the validation mean that we'll enforce the user to add preload attribute to amp-video? Cause I am not sure if that's the goal. 

This will propagate the overridden value ('none' in the case of `prerender` mode and the user-provided value otherwise), since I update the preload value on the element itself the propagation will have the correct value.